### DISCUSSION TITLE
Actions & Weird behaviour.

### DISCUSSION THREAD

**Discussion by Olzie-12** [community/community#112296] [Actions]
### Select Topic Area

Bug

### Body

GitHub actions, even stated they’re working fine. They’re not currently working.
Webhooks and API calls are also on the edge as well.

Had the exact same issue yesterday, came back and it was still broken.

**Comment by jimmybsilva** [comment]
Did your actions stop triggering? My actions are not triggering?

**Reply by GeoffreyChen777** [reply]
Same!!!! What's going on!!

**Reply by jimmybsilva** [reply]
Yea, I tried everything so I guess its not just me. It had been working for months with no issue and today It stopped triggering. Not sure where I have to report it. https://www.githubstatus.com/ shows no issues with actions maybe it just started happening.

**Reply by GeoffreyChen777** [reply]
I'm going to release a new version tonight, but I can't trigger the build anyway. I spent more than an hour checking the workflow file. 😵

It's time to go to bed.

**Reply by jimmybsilva** [reply]
Lol if it were not for this discussion I probably would have tried playing with it for a few hours..lol Thanks [Olzie-12] for posting

**Reply by dirkhh** [reply]
same here - tried 20 different things, deleted tags, pushed them again, looked at GitHub Status (which claims that Actions are working)... glad I'm not alone

**Reply by Sn0w3y** [reply]
Phew.... i am glad i am reading this.. been searching for hours :D

**Reply by chriswbartley** [reply]
Me too - actions are not running on commits as of today...

**Reply by dirkhh** [reply]
And ironically, all the different things that I tried and that each should have triggered an action have now indeed caused quite a few actions to run 🤷‍♂️

**Comment by Olzie-12** [comment]
Yeah we hooks ain’t working for me either, actions, webhooks, commits and api calls

**Reply by jimmybsilva** [reply]
I submitted a ticket and hope they get back to me soon.

**Comment by turfaa** [comment]
Same. My Actions stopped triggering. I was confused because githubstatus.com says the issue only happens in PRs, not Actions.

**Reply by jimmybsilva** [reply]
It looks like GitHub is experiencing issues specifically related to pull requests and possibly their underlying infrastructure, which could be affecting GitHub Actions as well. When there's degraded performance or other issues affecting pull requests, it's reasonable to assume that related features like GitHub Actions might also be impacted, either directly or due to increased load or instability in the system.

**Comment by Sn0w3y** [comment]
> I'm going to release a new version tonight, but I can't trigger the build anyway. I spent more than an hour checking the workflow file. 😵
> 
> It's time to go to bed.


Tried 10 YAML-Validators, tried Dummy PRs :D searched for hours for Issues on my Side - same same

**Comment by orange723** [comment]
I push dev branch action is not run , I don't know think .
Use workflow_dispatch fix this issue but is hand action.
When the fix action issue?
Looks like I'm not the only one with this problem.

**Comment by chriswbartley** [comment]
My actions just started triggering again...

**Reply by jimmybsilva** [reply]
same here

**Comment by orange723** [comment]
Delay action is running now

**Comment by Olzie-12** [comment]
All actions are now back up and running 👍🏼
